### PR TITLE
Changed variable names to fix CI

### PR DESCRIPTION
**Pull request by mpatsute**
cc @bajtos @raymondfeng @ritch @superkhau
This PR is for fixing the CI. The CI was failing with the following error, so changed some variable names.  (FYI. This is a blocker for my other PR: https://github.com/strongloop/loopback-next/pull/413)
```
ERROR: /home/jenkins/workspace/nb/loopback-next~master/7a48ad0e/packages/authentication/test/acceptance/basic-auth.ts[204, 21]: Shadowed name: 'key'
ERROR: /home/jenkins/workspace/nb/loopback-next~master/7a48ad0e/packages/authentication/test/acceptance/basic-auth.ts[130, 45]: Shadowed name: 'getFromContext'
ERROR: /home/jenkins/workspace/nb/loopback-next~master/7a48ad0e/packages/authentication/test/acceptance/basic-auth.ts[134, 42]: Shadowed name: 'bindElement'
ERROR: /home/jenkins/workspace/nb/loopback-next~master/7a48ad0e/packages/authentication/test/acceptance/basic-auth.ts[136, 19]: Shadowed name: 'authenticate'
ERROR: /home/jenkins/workspace/nb/loopback-next~master/7a48ad0e/packages/repository/examples/controllers/customer-controller-with-constructor-di.ts[17, 13]: Shadowed name: 'repository'
```

### CODE REVIEW

**Comment by raymondfeng** [comment]
The failure seemed to be a violation of `no-shadowed-variable` tslint rule. Not sure why it shows up now.

File: packages/authentication/test/acceptance/basic-auth.ts [hunk 1]
```diff
@@ -127,24 +127,24 @@ describe('Basic Authentication', () => {
     class MySequence implements SequenceHandler {
       constructor(
         @inject('findRoute') protected findRoute: FindRoute,
-        @inject('getFromContext') protected getFromContext: getFromContext,
+        @inject('getFromContext') protected getFromCntxt: getFromContext,
```
Comment (raymondfeng) [inline packages/authentication/test/acceptance/basic-auth.ts]: IIRC, the property names should not be changed. Can we find out why they are shadowed?

File: packages/repository/examples/controllers/customer-controller-with-constructor-di.ts [hunk 2]
```diff
@@ -14,6 +14,6 @@ export class CustomerController {
   constructor(
     // Use constructor dependency injection
     @repository('Customer', 'mongodbDataSource')
-    private repository: EntityCrudRepository<Customer, string>,
+    private repo: EntityCrudRepository<Customer, string>,
```
Comment (raymondfeng) [inline packages/repository/examples/controllers/customer-controller-with-constructor-di.ts]: You need to fix refs to this property too

**Comment by bajtos** [comment]
FWIW, this problem is caused by tslint. The version 5.4.3 works fine, the latest 5.5.0 reports these warnings.

File: packages/authentication/test/acceptance/basic-auth.ts (hunk 1 above)
Comment (bajtos) [inline packages/authentication/test/acceptance/basic-auth.ts]: `getFromContext` is a type imported from `internal-types.ts`. At the same time, it's a property name defined on `MySequence` class. I'll fix this issue by renaming the type to `GetFromContext`, which is consistent with other existing type names like `FindRoute`.

**Comment by bajtos** [comment]
I reworked the pull request. I think fixing the type names to `GetFromContext` and `BindElement` make a lot of sense. The other changes may make less sense, perhaps we may want to consider disabling `no-shadowed-variable` completely?

@raymondfeng @superkhau PTAL again

**Comment by superkhau** [comment]
@slnode test please

**Comment by mpatsute** [comment]
Merging the PR

**Pull request merged**